egs/librispeech/s5/local/chain/run_chain_common.sh

```diff
@@ -3,133 +3,76 @@
 # this script has common stages shared across librispeech chain recipes
 set -e
 
-# configs for 'chain'
-stage=0
-# chain options
-frames_per_eg=150
-max_wer=
+stage=11
+
+# input directory names
+gmm_dir=
```

Comment:
Say that those these options are being provided  as named options they are actually compulsory and that they have been named for convenience.